src/universal/redux/segmentActions.js

```diff
@@ -1,22 +1,53 @@
 import {EventTypes} from 'redux-segment';
+import {selectProfile} from './profileDuck';
 
 /**
  * Sometimes, we just want to track events in segment.
  */
 
 const SEGMENT_EVENT = '@@segment/EVENT';
 
-export function segmentEvent(event, properties) {
-  return {
-    type: SEGMENT_EVENT,
-    meta: {
-      analytics: {
-        eventType: EventTypes.track,
-        eventPayload: {
-          event,
-          properties
+export function segmentEventTrack(event, properties, options) {
+  return (dispatch, getState) => {
+    const profile = selectProfile(getState());
```

Comment:
`preferredName = name` & `avatar image = picture`, right? so i think the fields are the same (and we can always request more if needed).
The `User` is persisted to localStorage & is only wiped on logout.
We know the `userId = auth.obj.sub`. So `me = state.cashay.entities.User[state.auth.obj.sub]`